src/index.js

```diff
@@ -9,6 +9,7 @@
  * - See {@link ws-date-picker/ws-date-picker.md|WSDatePicker}
  * - See {@link ws-dropdown/ws-dropdown.md|WSDropdown}
  * - See {@link ws-notification/ws-notification|WSNotification}
+ * - See {@link ws-week-picker/ws-week-picker|WSWeekPicker}
```

Comment:
I'm sorry, could you please change this to {@link ws-week-picker/ws-week-picker.md|WSWeekPicker} with the ".md" Extension